lib/index.d.ts

```diff
@@ -2,7 +2,7 @@
 import BN from 'bn.js';
 export declare function baseEncode(value: Uint8Array | string): string;
 export declare function baseDecode(value: string): Buffer;
-export declare type Schema = Map<Function, any>;
```

Comment:
Curious - why did we remove this `declare` keyword?

Reply:
that was the automatic build, I can manually return it, no problem.